ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml

```diff
@@ -1906,6 +1906,38 @@ tests:
     test:
     - ref: openshift-extended-test
     workflow: ipi-openstack
+- as: azure-ipi-fips-p3-f2-openshift-logging-5-5
+  cron: 00 21 * * 1,3,5
```

Comment:
I'm thinking if it's too frequently. Maybe we can add more profiles and reduce the frequency of each profile. @IshwarKanse @anpingli any thoughts?

Reply:
@QiaolingTang I agree, instead of running the same profile 3 times in a week, we can use different profiles. Currently we can keep the profiles to 3-4 so that it doesn't become overwhelming for us Let me know if you have any suggestions for the profiles which we can add which would make these job runs valuable.

Reply:
ARM, HTTP_PROXY, global proxy, STS we can consider these.

Reply:
Yes. How about finish all these test in 3 days.